### ISSUE TITLE
Why is the result different from the demo?

### ISSUE THREAD

**Issue by soohyoen** [NVlabs/InstantSplat#74]
Thank you for your great work, I got good results by inputting 8 images on your demo page.
However, when I operate your code directly, the results are very different.
Artifact looks worse when turned into code than demo
What is the problem?

- Code
<img width="1822" alt="Image" src="https://github.com/user-attachments/assets/c5dec6a4-53ba-45b5-bcf5-189eb5aa2bd6" />

- Demo
<img width="1822" alt="Image" src="https://github.com/user-attachments/assets/3c7beca1-d0b8-4ee7-a775-35400eaff90e" />

**Comment by kairunwen** [comment]
Hi, The demo employs lower initial learning rates, while the [code](https://github.com/NVlabs/InstantSplat/blob/main/scene/gaussian_model.py#L206-L213) sets higher values for parameters like `xyz_lr` and `feature_lr`. The difference may stem from these initial rates. Try lowering the rates in training_setup.

**Comment by soohyoen** [comment]
@kairunwen 
Thank you so much for your quick response. Can you tell me how much difference there is?

**Comment by kairunwen** [comment]
10 times.

**Comment by soohyoen** [comment]
@kairunwen 

Hello, thank you so much for your kind response.

However, the problem is not solved yet. 
I reduced learning_rate by about 10, but the result is as shown in the picture.
I think it has a soft shape when I see the results of demo.

What I want is things that have less artifact in the gaussian form like I ran in demo.

Is there an answer other than learning_rate?

- code
![Image](https://github.com/user-attachments/assets/3681e45f-aea2-4bdc-8587-60145602e500)

- demo

![Image](https://github.com/user-attachments/assets/7e6c2cc0-6622-44c0-8c16-2ed2ef45d6ee)

**Comment by starwalkin-C** [comment]
@soohyoen Can you inspect the result of your rendering from `render.py` and see if the result is consistent with your visualization?

**Comment by soohyoen** [comment]
@kairunwen @starwalkin-C 

The results after render.py look neat.
But they're not neat when you see them on superplat.
Is the instantsplat-2dgs on demo?

**Comment by kairunwen** [comment]
@soohyoen Hi, could you provide the image corresponding to this Gaussian? I need it to debug this issue.

**Comment by soohyoen** [comment]
@kairunwen 
Thank you so much for your answer

Attached is the picture you rendered after learning.


- Train view
![Image](https://github.com/user-attachments/assets/4a6bc42f-f3b1-4ff0-a175-16be4be2c712)

![Image](https://github.com/user-attachments/assets/7b467ab3-b380-47b4-9ce9-72d81597c495)

- Test view

![Image](https://github.com/user-attachments/assets/e1ffb9e5-2dd7-4f89-87d2-245a763e68b3)

![Image](https://github.com/user-attachments/assets/a44e6e70-ed7d-4f42-9f06-54a1409ce9cf)


There were eight train views and six test views and a few were sampled.

But what's a little strange is that the first picture continues from 1 to 7 in the train view I attached.

If you need additional information, please request it

**Comment by kairunwen** [comment]
@soohyoen Hi, could you send all images to my email~: wenkairun@gmail.com.

**Comment by kairunwen** [comment]
Hi @soohyoen, after debugging on my own data, I found that this issue is indeed caused by a high learning rate (lr). You can add some constraints on the scale or simply lower the learning rate to address it. (Reference: https://github.com/NVlabs/InstantSplat/issues/42#issuecomment-2564630678).

If you could share your original images to my email: wenkairun@gmail.com, it would greatly help with further debugging.
